client/hawc_client/summary.py

```diff
@@ -49,3 +54,115 @@ def table_list(self, assessment_id: int) -> pd.DataFrame:
         url = f"{self.session.root_url}/summary/api/summary-table/?assessment_id={assessment_id}"
         response_json = self.session.get(url).json()
         return pd.DataFrame(response_json)
+
+    def download_visuals(
+        self, assessment_id: int, dest: str = ".", file_type: str = "docx", headless: bool = True
+    ):
+        """
+        Retrieves all visuals for the given assessment and saves them in the given path.
+        Session must be authenticated with a token to access private assessment visuals.
+
+        Args:
+            assessment_id (int): Assessment ID
+            dest (str): Folder where images will be saved. Defaults to current working directory.
```

Comment:
If we could make this either a string or a `bytes.IO` instance, then we could write to disk if string, and use memory if `bytes.IO`. That would fix the temp file directory issue too I think